src/functions.date.js

```diff
@@ -94,6 +94,20 @@ $.extend(_, {
 	}, {multiValued: true}),
 
 	localTimezone: -(new Date()).getTimezoneOffset(),
+	getduration: $.extend(function (dur, format) {
```

Comment:
It's unclear what the purpose of `getduration()` is, or how reusable it is. It seems to take a formatted duration string, and return a different formatted duration string. I would suggest decoupling it into a function that parses a duration string and returns a number, and then we already have a function that takes a number and converts it to a formatted duration string (`duration()`).

A few more minor things:

* Avoid abbreviated names like `dur`
* inconsistent naming, camelCase in the line above, lowercase concatenated in this one. In general, JS tends to use camelCase, and it's usually good to follow the naming conventions of the host language.
* `Object.assign()` should work fine instead of `$.extend()` here, the main difference being that the former executes accessors and assigns the result, whereas the latter transfers the accessors. When there are no accessors, `Object.assign()` is preferable as it's native to the platform, and thus faster.